### DISCUSSION TITLE
More level colours

### DISCUSSION THREAD

**Discussion by JamesSiebert** [opcodesio/log-viewer#203] [Ideas]
Hi, Thanks for this log-viewer, its the best! :D

I'm wondering if there is a way to add a green success log entry or add the possibility to make this happen
Even if its something as simple as make notice show as green instead of blue.

Best case would be a 'success' log category but just having one of the types as green would be a good start.

Have a great day and keep up the awesome work! :D

**Comment by arukompas** [comment]
Hey @JamesSiebert 
Thanks for your support, I'm glad you like it! 😀 

According to PSR-3, there is no "success" log level - https://www.php-fig.org/psr/psr-3/

Also there's no such method in Laravel like `Log::success(...)`

Can you share an example of how you're getting such a log entry, and how it's represented in your log file?

**Comment by JamesSiebert** [comment]
It honestly makes tracking things so much easier, I use it daily :D

Currently I'm just using notice or info as my success type, It would be great if they had different colours to easily differentiate them at a glance. e.g  info=blue, notice=green, debug=grey.
Currently notice info and debug are all the same blue. 

Another option, which you probably have a better way of doing this, but what if there was a way to set these colours?
- Config file - info = #AEEEEE
- or Short code - if the log contains: [log-viewer-green]
- or with contextual info: https://laravel.com/docs/10.x/logging#contextual-information

**Reply by arukompas** [reply]
I really like your suggestions! I'll keep this in mind for the next release :D At the moment I'm busy working on multiple-hosts and accessibility features, so maybe after I'm done with those :)

**Reply by JamesSiebert** [reply]
Thank you :D 
Have a great day!
